src/main/java/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,40 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum LottoRank {
+    RANK_FIFTH(3, false, 5000),
```

Comment:
이런 숫자들의 경우는 아래와 같이 언더바(_)를 사용하여 가독성을 높힐 수 있어요! 

```suggestion
    RANK_FIFTH(3, false, 5_000),
```

읽는이를 배려하는 코드로 리팩터링 해볼까요?